datafusion/core/src/execution/session_state.rs

```diff
@@ -177,23 +177,23 @@ impl Debug for SessionState {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.debug_struct("SessionState")
             .field("session_id", &self.session_id)
-            .field("analyzer", &"...")
+            .field("config", &self.config)
```

Comment:
I wonder if we could simply `#derive(Debug)` for SessionState now? 

And perhaps we could provide a `impl Display for SessionState` that had a more readable format (like that shows session_id and deviation from default configuration, for example 🤔 )